app/controllers/admin/creator_settings_controller.rb

```diff
@@ -12,13 +13,13 @@ def create
         ::Settings::UserExperience.primary_brand_color_hex = settings_params[:primary_brand_color_hex]
         ::Settings::Authentication.invite_only_mode = settings_params[:invite_only]
         ::Settings::UserExperience.public = settings_params[:public]
+        settings_params[:checked_code_of_conduct] = true
+        settings_params[:checked_terms_and_conditions] = true
```

Comment:
```suggestion
        checked_code_of_conduct = settings_params[:checked_code_of_conduct]
        checked_terms_and_conditions = settings_params[:checked_terms_and_conditions]
```

Once you apply the change to the HTML, these values come through fine and your backend should work consistently.

Reply:
Thanks so much for the help, @Ridhwana! I've applied the suggested changes, but I'm still running into issues with the backend:
- rubocop is unhappy with the following and complains that the following assignment is useless:
```
 checked_code_of_conduct = settings_params[:checked_code_of_conduct]
        checked_terms_and_conditions = settings_params[:checked_terms_and_conditions]

```
- the `checked_code_of_conduct` and `checked_terms_and_conditions` values don't actually update on the `current_user`, as shown below, thus throwing the `current_user` into the user onboarding after clicking the "Finish" button:
```
[1] pry(#<Admin::CreatorSettingsController>)> current_user
=> #<User id: 179, apple_username: nil, articles_count: 0, badge_achievements_count: 0, blocked_by_count: 0, blocking_others_count: 0, checked_code_of_conduct: false, checked_terms_and_conditions: false
```

Reply:
OOOh i didn't notice it was in a `ActiveRecord::Base.transaction`, that's probably the reason. Looking into this in a bit.